k8s/k8s.go

```diff
@@ -930,6 +934,15 @@ func (k *k8sOps) DescribeDeployment(depName, depNamespace string) (*apps_api.Dep
 	return &dep.Status, err
 }
 
+func (k *k8sOps) UpdateDeployment(deployment *apps_api.Deployment) (*apps_api.Deployment, error) {
+	logrus.Info("Attempt to update the deployment.")
+	dep, err := k.appsClient().Deployments(deployment.Namespace).Update(deployment)
+	if err != nil {
```

Comment:
don't check error and return, just return the output as is
`return k.appsClient().Deployments(deployment.Namespace).Update(deployment)`
this is much simpler and the same thing that you are doing.